doc: lwm2m: Announces support for v.1.1 (#48687)
zephyrproject-rtos/zephyr: doc/connectivity/networking/overview.rst

  (`LwM2M specification 1.0.2`_) is supported via the "Bootstrap", "Client
  Registration", "Device Management & Service Enablement" and "Information
  Reporting" interfaces.  The required core LwM2M objects are implemented as

>   well as several IPSO Smart Objects. (`LwM2M specification 1.1.1`_) is
>   supported in similar manner when enabled with an overlay file.

Reviewer: ```suggestion
  well as several IPSO Smart Objects. (`LwM2M specification 1.1.1`_) is
  supported in similar manner when enabled with a Kconfig option.
```
